src/cli.rs

```diff
@@ -67,68 +71,80 @@
 //! Product config handling works similarly:
 //!
 //! ```no_run
-//! use clap::{crate_version, SubCommand};
+//! use clap::{crate_version, App, Parser};
 //! use stackable_operator::cli;
 //! use stackable_operator::error::OperatorResult;
-//! use clap::App;
+//!
+//! #[derive(clap::Parser)]
+//! #[clap(
+//!     name = "Spark Operator",
+//!     author,
+//!     version,
+//!     about = "Stackable Operator for Foobar"
```

Comment:
Same here

Reply:
Switched the remaining names to "Foobar Operator".